src/ControllerV2/ProController.php

```diff
@@ -13,26 +15,74 @@
 use App\ServiceV2\PaginatorService;
 use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 #[Route(path: '/pro')]
+#[IsGranted(ProVoter::MANAGE)]
 class ProController extends AbstractController
 {
+    #[Route(path: '', name: 'list_pro', methods: ['GET'])]
+    public function listPros(Request $request, MembreRepository $repository, PaginatorService $paginator): Response
+    {
+        return $this->render('v2/pro/list/professionals.html.twig', [
+            'professionals' => $paginator->create(
+                $repository->findByAuthorizedProfessional($this->getUser()->getSubject()),
+                $request->query->getInt('page', 1),
+                PaginatorService::LIST_USER_LIMIT,
+            ),
+            'form' => $this->createForm(FilterUserType::class, null, [
+                'action' => $this->generateUrl('filter_pro'),
+                'attr' => ['data-controller' => 'ajax-list-filter'],
+                'relays' => $this->getUser()->getAffiliatedRelaysWithProfessionalManagement(),
+            ]),
+        ]);
+    }
+
+    #[Route(
+        path: '/filter',
+        name: 'filter_pro',
+        methods: ['POST'],
+        condition: 'request.isXmlHttpRequest()',
+    )]
+    public function filterPros(Request $request, MembreRepository $repository, PaginatorService $paginator): Response
+    {
+        $formModel = new FilterUserFormModel();
+        $form = $this->createForm(FilterUserType::class, $formModel, [
+            'action' => $this->generateUrl('filter_pro'),
+            'attr' => ['data-controller' => 'ajax-list-filter'],
+            'relays' => $this->getUser()->getAffiliatedRelaysWithProfessionalManagement(),
+        ])->handleRequest($request);
+
+        return new JsonResponse([
```

Comment:
I am not sure yo need to send a JsonResponse, you can also send a classic response, but you need to catch all the response body in your frontend